website/src/pages/hackathons.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import Helmet from 'react-helmet'
+import Layout from '../components/Layout'
+import { eventLogger } from '../EventLogger'
+
+export default class Pricing extends React.Component<any, any> {
+    public componentDidMount(): void {
+        const script = document.createElement('script')
+        script.src = '//js.hsforms.net/forms/v2.js'
+        const hubspot = document.getElementById('hubspotTrialForm')
+        hubspot.appendChild(script)
+        script.addEventListener('load', () => {
+            ;(window as any).hbspt.forms.create({
+                portalId: '2762526',
+                formId: '7d6c55af-3de3-4e57-a5df-a0de341a4814',
+                target: '#hubspotTrialForm',
+                onFormSubmit: (e: Event) => {
+                    eventLogger.trackHackathonFormSubmitted()
+                },
+            })
+        })
+    }
+
+    public render(): JSX.Element | null {
+        return (
+            <Layout location={this.props.location}>
+                <div>
+                    <Helmet>
+                        <title>Sourcegraph - Set up Sourcegraph at a Hackathon</title>
+                        <meta name="twitter:title" content="Set up Sourcegraph at a Hackathon" />
+                        <meta property="og:title" content="Set up Sourcegraph at a Hackathon" />
+                    </Helmet>
+                    <div className="pricing-bg" />
+                    <div className="jumbotron text-right dark-7">
+                        <div className="row">
+                            <div className="sales py-0 d-flex flex-grow flex-column flex-lg-row align-items-center justify-content-around">
+                                <div>
+                                    <h1 className="text-center">Set up Sourcegraph during a hackathon</h1>
+                                    <p className="text-center measure">
+                                        Hackathons are the perfect time to set up code search and intelligence for your
+                                        team. We'll give you access to all of Sourcegraph's features to set you up for
+                                        the best possible demo.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <section className="d-flex justify-content-around flex-column flex-lg-row sales bg-white">
+                        <div className="panel-area mr-lg-5 mt-0 mb-2">
+                            <div className="d-flex flex-column panel">
+                                <p>
+                                    Want to bring lasting value to your company during a hackathon? Set up Sourcegraph
```

Comment:
This is great, but I've never heard this wording before. It sounds a bit serious/formal to me ("lasting value") — maybe something fun/flippant like "Want to win your Hackathon?"